.github/workflows/test.yml

```diff
@@ -52,8 +52,8 @@ jobs:
     - name: Install Distribution
       run: poetry run poe install_dist
 
-    - name: Remove Source Code
-      run: poetry run poe remove_src
+    - name: Rename Source Code
+      run: poetry run poe rename_src
```

Comment:
Optional: would be nice to have it as a context manager

with rename_src():
    tests()

Reply:
I did not get it, could you explain more details?